src/edu/csus/ecs/pc2/clics/API202306/CLICSLocation.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * CLICS geographic location
+ * Contains information the location of another object
+ * 
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CLICSLocation {
+    private static final double MAX_LATITUDE = 90.0;
+    private static final double MAX_LONGITUDE = 180.0;
+
+    @JsonProperty("latitude")
+    private double latitude = MAX_LATITUDE + 1;
+    
+    @JsonProperty("longitude")
+    private double longitude = MAX_LONGITUDE + 1;
+
+    /**
+     * Fill in location information properties
+     * 
+     * @param latitude The location's latitude
+     * @param longitude The location's longitude
+     */
+    public CLICSLocation(double latitude, double longitude) {
```

Comment:
Since the CLICS spec explicitly states that latitude ranges from -90..90 and longitude from -180..180, it seems like this constructor should check the input values and generate an error (throw an `IllegalArgumentException`).  Otherwise we are tacitly supporting the creation of an invalid `CLICSLocation` object.   (And I think the error-checking belongs here, in the constructor; we can't assume that all existing _and future_ code will edit-check the data it passes to the constructor.)

Reply:
Agree. Added.  Not much to do but throw an IllegalArgumentException and let the caller worry about. Added in next push.